src/App.js

```diff
@@ -1,20 +1,34 @@
-import React from "react";
+import React, { Component } from "react";
 import "./App.css";
-import StyledButton from "./stories/Button";
 
-import { Placeholder } from "./stories/Input/component.stories";
+import { Route, Switch, withRouter, Redirect } from "react-router-dom";
 
-function App() {
-    return (
-        <div className="App">
-            {/* To be updated */}
-            <form>
-                <h1>Magical List</h1>
-                <Placeholder />
-                <StyledButton>Submit</StyledButton>
-            </form>
-        </div>
-    );
+import Layout from "./hoc/Layout/Layout";
+import Login from "./containers/Authentication/Login";
+import Register from "./containers/Authentication/Register/Index";
+import Home from "./containers/Tasks/Home";
+
+class App extends Component {
+    render() {
+        let routes = (
+            <Switch>
+                <Route path="/login" component={Login}>
+                    Login
+                </Route>
+                <Route path="/signup" component={Register}>
+                    Register
+                </Route>
+                <Route path="/home" component={Home} />
+
+                <Redirect to="/" />
+            </Switch>
+        );
+        return (
+            <div className="App">
+                <Layout>{routes}</Layout>
```

Comment:
Handle Auth conditional logic inside this component, it is not very clear that Layout actually does this.

Reply:
done